README.md

```diff
@@ -1,33 +1,355 @@
+        - `OME-TIFF`
+        - `OME-ZARR`
+        - `PNG`, `GIF`, [and other similar formats seen here](https://github.com/imageio/imageio)
+- Various File Systems (HTTP URLs, s3, gcs, etc.)
+    - Supports reading and writing to [fsspec](https://github.com/intake/filesystem_spec) supported file system wherever possible:
+        -   Local paths (i.e. `my-file.png`)
+        -   HTTP URLs (i.e. `https://my-domain.com/my-file.png`)
+        -   [s3fs](https://github.com/dask/s3fs) (i.e. `s3://my-bucket/my-file.png`)
+        -   [gcsfs](https://github.com/dask/gcsfs) (i.e. `gcs://my-bucket/my-file.png`)
+    - See [Cloud IO Support](#cloud-io-support) for more details.
+
 ## Installation
 
 **Stable Release:** `pip install bioio`<br>
 **Development Head:** `pip install git+https://github.com/bioio-devs/bioio.git`
 
+BioIO is supported on Windows, Mac, and Ubuntu.
+For other platforms, you will likely need to build from source.
+
+### Reader Installation
+
+BioIO is a plug-in based system of readers meaning in addition to the `bioio` package you need to install the packages
+that support the file types you are using. For example, if attempting to read `.ome.tiff` and `.zarr` files you'll want to install the `bioio-ome-tiff` & `bioio-ome-zarr` packages alongside `bioio` (ex. `pip install bioio bioio-ome-zarr bioio-ome-tiff`).
+BioIO will then determine which reader to use for which file automatically.
+
+This is a list of currently known and maintained reader plug-ins available:
+| Package      | Supported file types |
+| ------------ | -------------------- |
+| [bioio-czi](https://github.com/bioio-devs/bioio-czi) | `CZI` |
+| [bioio-dv](https://github.com/bioio-devs/bioio-dv)  | `DV` |
+| [bioio-imageio](https://github.com/bioio-devs/bioio-imageio) | `PNG`, `GIF`, [& other similar formats seen here](https://github.com/imageio/imageio)        |
+| [bioio-lif](https://github.com/bioio-devs/bioio-lif) | `LIF`        |
+| [bioio-nd2](https://github.com/bioio-devs/bioio-nd2) | `ND2`        |
+| [bioio-ome-tiff](https://github.com/bioio-devs/bioio-ome-tiff) | `OME-TIFF` (non-tiled)      |
+| [bioio-ome-tiled-tiff](https://github.com/bioio-devs/bioio-ome-tiled-tiff) | `OME-TIFF` (tiled)       |
+| [bioio-ome-zarr](https://github.com/bioio-devs/bioio-ome-zarr) | `ZARR`        |
+| [bioio-sldy](https://github.com/bioio-devs/bioio-sldy) | `SLDY`        |
+| [bioio-tifffile](https://github.com/bioio-devs/bioio-tifffile) | `TIFF` (non-globbed)       |
+| [bioio-tiff-glob](https://github.com/bioio-devs/bioio-tiff-glob) | `TIFF` (globbed)       |
+| [bioio-bioformats](https://github.com/bioio-devs/bioio-bioformats) | Files supported by [Bio-Formats](https://docs.openmicroscopy.org/bio-formats/latest/supported-formats.html) (Requires `java` and `maven`, see below for details)       |
+
+## Documentation
+
+For full package documentation please visit
+[https://bioio-devs.github.io/bioio/](https://bioio-devs.github.io/bioio/).
```

Comment:
this is self referential in the published docs site and this `#Documentation` section could just be removed.

Reply:
Continuing this conversation in another thread, marking this thread as resolved